tests/apps/verify-ppb/pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[tool.briefcase]
+project_name = "Hello PPB"
+bundle = "org.beeware"
+version = "0.0.1"
+url = "https://beeware.org/"
+license = "BSD license"
+author = 'Brutus'
+author_email = "contact@beeware.org"
+
+[tool.briefcase.app.verify-ppb]
+template = "../../.."
```

Comment:
This should probably be in the `[macOS]` block, to avoid any possible confusion.

Reply:
Done (and renamed)